source/gpgpu.rst

```diff
@@ -0,0 +1,30 @@
+.. toctree::
+
+GPGPUs on EGI Cloud
+===================
+
+GPGPUs resources are available on selected providers of the EGI Cloud. These 
+are available as specific **flavors** that when used to instantiate a Virtual
+Machine will make the hardware available for the user.
+
+The table below summarises the available options:
+
++-------------------+---------------------------------+--------------------------------------------+
+| Site              | VM configuration options        | Access conditions                          |
++===================+=================================+============================================+
+| **IISAS-GPU**     | up to 2 NVIDIA Tesla K20m       | Sponsored access for limited testing,      |
+|                   |                                 | conditions to be negotiated for long-term  |
+|                   |                                 | usage                                      |
++-------------------+---------------------------------+--------------------------------------------+
+| **IISAS-Nebula**  | up to 2 NVIDIA Tesla K20m       | Sponsored access for limited testing,      |
+|                   |                                 | conditions to be negotiated for long-term  |
+|                   |                                 | usage                                      |
++-------------------+---------------------------------+--------------------------------------------+
+| **IFCA-LCG2**     | up to 4 Nvidia GT200GL          | Pay-per-use                                |
+|                   +---------------------------------+                                            |
+|                   | up to 1 Nvidia Titan X          |                                            |
+|                   +---------------------------------+                                            |
+|                   | up to 10 Nvidia Titan 1080ti    |                                            |
```

Comment:
Not sure Nvidia Titan 1080 Ti exist, isn't it common GeForce like https://www.nvidia.com/en-us/geforce/products/10series/geforce-gtx-1080-ti/ ?

Reply:
got the information directly from provider, so didn't really check. Also I'm realising capitalisation of NVIDIA is wrong.

Reply:
Should be fixed now.
Still the approval of the PR depends mostly on the providers giving green light. Contacted them via email